.rhcicd/clowdapp-backend.yaml

```diff
@@ -315,6 +317,9 @@ parameters:
 - name: INTERNAL_RBAC_ENABLED
   description: Enable Internal RBAC
   value: "false"
+- name: INTERNAL_RBAC_ADMIN
+  description: Internal admin group
+  value: "crc-notifications-team"
```

Comment:
I'm adding the parameter to be able to test on stage with a non admin user.

Reply:
This may be deployed on prod unintentionally, is that OK to commit that value?

Reply:
That's the default admin group. It should match what we have already in the properties file